src/components/BottomSheet/BottomSheet.js

```diff
@@ -39,14 +39,16 @@ const styles = StyleSheet.create({
   container: {
     paddingTop: dimen.normalizeDimen(10),
     paddingHorizontal: dimen.normalizeDimen(20),
-    paddingBottom: dimen.normalizeDimen(20)
+    paddingBottom: dimen.normalizeDimen(20),
+    backgroundColor: COLORS.almostBlack
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a background color to the `container` style is a good change, as it ensures that the container has a consistent background across different screens. However, please ensure that `COLORS.almostBlack` is accessible and provides sufficient contrast with any text or elements that may be placed on top of it.



<!-- This is an auto-generated comment by OSS CodeRabbit -->